app/src/main/shared/actions/AbstractPageActionCreators.ts

```diff
@@ -6,6 +6,10 @@ export abstract class AbstractPageActionCreators<TState, TPageProps> {
 
     abstract parent: AbstractPageActionCreators<TState, any>;
 
+    title(state: TState): string {
```

Comment:
I've not made this abstract so that it doesn't require us to change every inherited class.

Reply:
Agreed :)